Cargo.toml

```diff
@@ -35,7 +35,7 @@ required-features = ["uefi_bin"]
 
 [dependencies]
 xmas-elf = { version = "0.8.0", optional = true }
-x86_64 = { version = "0.14.7", optional = true, default-features = false, features = ["instructions", "inline_asm"] }
+x86_64 = { version = "0.14.9", optional = true, default-features = true, features = ["instructions", "inline_asm", "step_trait"] }
```

Comment:
Why are you enabling the default features?

Reply:
Huh, not sure why I did that, I reverted that particular change.